static/js/shared.js

```diff
@@ -77,3 +78,8 @@ exports.generateCommentId = function(){
 exports.generateReplyId = function(){
   return REPLY_PREFIX + randomString(16);
 }
+
+exports.isCommentsEnabledOnThisPadType = function() {
+  var padType = pad.plugins.ep_script_elements.padType.getPadTypeParam();
+  return padType !== epSEShared.TITLE_PAGE_DOCUMENT_TYPE;
+}
```

Comment:
o shared é chamado por client e server. Isso é uma convenção nossa, se isso for chamado pelo server acho deve dar erro pq n tem o `pad.plugins`. Melhor mover

Reply:
ok!